ccd-definition/CaseField.json

```diff
@@ -25,11 +25,28 @@
     "FieldType": "Text",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "EPO_REASONING_SHOW",
+    "Label": "EPO Reason show or hide",
+    "FieldType": "MultiSelectList",
```

Comment:
Does it have to be multi select list? What if we made it `Text`?

Reply:
Doesn't work with Text

Reply:
I hoped that CONTAINS works with strings.

Disgusting workaround :(